ArduSub/mode_surface.cpp

```diff
@@ -6,8 +6,8 @@ bool ModeSurface::init(bool ignore_checks)
     nobaro_mode = !sub.control_check_barometer();
 
     // initialize vertical speeds and acceleration
-    position_control->set_max_speed_accel_U_cm(-sub.get_pilot_speed_dn(), g.pilot_speed_up, g.pilot_accel_z);
-    position_control->set_correction_speed_accel_U_cm(-sub.get_pilot_speed_dn(), g.pilot_speed_up, g.pilot_accel_z);
+    position_control->set_max_speed_accel_U_cm(sub.get_pilot_speed_dn(), g.pilot_speed_up, g.pilot_accel_z);
+    position_control->set_correction_speed_accel_U_cm(sub.get_pilot_speed_dn(), g.pilot_speed_up, g.pilot_accel_z);
```

Comment:
```suggestion
    position_control->set_correction_speed_accel_V_cm(sub.get_pilot_speed_dn(), g.pilot_speed_up, g.pilot_accel_z);
```

please do consider `V` for vertical where we are passing in values where some are up and some are down.